style.css

```diff
@@ -33,6 +34,7 @@ hr {
 
 #layer {
   position: fixed;
+  backdrop-filter: blur(3px);
   height: 100%;
   top: 0;
   left: 0;
```

Comment:
- [ ] Nice work adding the blur effect in the background requested by the previous reviewer 👏 . But your close-icon seems to be gone now. Check if you have the relative path in your src code to that icon. And now, page has a vertical scroll which shouldn't be there. Kindly remove that scrolling: 
![image](https://user-images.githubusercontent.com/91486791/203615769-b0454227-4a39-4f35-acf4-f6244c318b9f.png)

Reply:
Thanks for the feedback. Both issues are resolved now. 😃